### DISCUSSION TITLE
Release for Kotlin 1.6

### DISCUSSION THREAD

**Discussion by theexiile1305** [jobrunr/jobrunr#268] [General]
Hey @rdehuyss 
thank you very much for this great project.

When do you plan to release the next version of jobrunr to fully support kotlin with version 1.6.0.  If you need support, I can be of help with pleasure.

Many greetings,
Michael

**Comment by rdehuyss** [comment]
Done

**Reply by theexiile1305** [reply]
Thank you very much!
